checkov/arm/checks/resource/AzureDefenderOnSqlServersVMS.py

```diff
@@ -0,0 +1,31 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.arm.base_resource_check import BaseResourceCheck
+from typing import List, Dict, Any
+
+
+class AzureDefenderOnSqlServersVMS(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that Azure Defender is set to On for SQL servers on machines"
+        id = "CKV_AZURE_79"
+        supported_resources = ("Microsoft.Security/pricings",)
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(
+            name=name,
+            id=id,
+            categories=categories,
+            supported_resources=supported_resources,
+        )
+
+    def scan_resource_conf(self, conf: Dict[str, Any]) -> CheckResult:
+        properties = conf.get("properties", {})
+        resourceType = properties.get("resourceType")
```

Comment:
I don't think there is `resourceType` in this resource, we can remove it.

Reply:
so how do we det to the "resourceType": "SqlServerVirtualMachines" to test it?

Reply:
we dont need it for ARM